### PR TITLE
Add sectionmotd aa1 to master

### LINKED ISSUE #18: There should be a graphical update to the motd
**Is your feature request related to a problem? Please describe.**
It would be nice, if the motd would get a graphical update, due to the motd not fitting in with the rest of the website.

**Describe the solution you'd like**
Update the look of the motd, so that it blends in with the rest of the website.

### PR DESCRIPTION
**Pull request by petschna**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the section message of the day(motd) does not fit in the current style of the website. #18

**What is the new behavior (if this is a feature change)?**
- Now there is a motd which has been updated to fit the style requirements. Closes #18

### CODE REVIEW

File: src/components/organisms/SectionMotd/index.jsx [hunk 1]
```diff
@@ -4,14 +4,18 @@ import React from "react";
...
 //> CSS
 import "./motd.scss";
 
+//> Video
+import SynthwaveVID from "../../../assets/content/synthwave/ezgif.com-crop.mov";
+
 //> Data
 const value = {
   motd: "Sky's The Limit",
+  //motd: "",
```
Comment (kleberbaum) [inline src/components/organisms/SectionMotd/index.jsx]:
```suggestion
```

File: src/components/organisms/SectionMotd/index.jsx [hunk 2]
```diff
@@ -20,9 +24,17 @@ class SectionMotd extends React.PureComponent {
...
           <a href="https://www.youtube.com/watch?v=d6a4hvoY0G0" target="_blank">
-            <h2>{value.motd}</h2>
+            <div class="motdh">{value.motd}</div>
           </a>
         </section>
+        
+        <video autoPlay muted loop className="fsynthtop">
+          <source src={SynthwaveVID} type="video/mp4" />
+        </video>
+
```
Comment (kleberbaum) [inline src/components/organisms/SectionMotd/index.jsx]:
```suggestion
```

**Review by petute — APPROVED** [review]
Should be ok to merge.

File: src/components/organisms/SectionMotd/index.jsx [hunk 3]
```diff
@@ -20,9 +24,17 @@ class SectionMotd extends React.PureComponent {
       <article class="motd">
         <section class="motd-s">
           <a href="https://www.youtube.com/watch?v=d6a4hvoY0G0" target="_blank">
-            <h2>{value.motd}</h2>
+            <div class="motdh">{value.motd}</div>
           </a>
         </section>
+
```
Comment (kleberbaum) [inline src/components/organisms/SectionMotd/index.jsx]:
```suggestion
```

File: src/components/organisms/SectionMotd/motd.scss [hunk 4]
```diff
@@ -1,13 +1,76 @@
...
+    z-index: 0;
+    bottom: 0px;
+    position: absolute;
+    height: 25%;
+    //height: 50%;
+    min-width: 100vw;
+    object-fit: fill;
+    filter: hue-rotate(292deg) saturate(180%) brightness(180%) contrast(84%) drop-shadow(0px -12px 12px rgb(233, 0, 241));
+  }
+}
```
Comment (kleberbaum) [inline src/components/organisms/SectionMotd/motd.scss:25]:
```suggestion
}

```

**Review by petschna — COMMENTED** [review]
Should be ok to merge.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**